docker-compose.yml

```diff
@@ -1,31 +1,35 @@
 version: '3.1'
 
 services:
-
   server:
     image: deviavir/zenbot:unstable
+    # build: .
     volumes:
-      - ./conf.js:/app/conf.js
-      - ./extensions:/app/extensions
-      - ./simulations:/app/simulations
+      - '.:/app'
     links:
       - mongodb
-    command: [ "trade", "--paper" ]
+    working_dir: /app
+    entrypoint: sh -c
+    command: ["./startup.sh"]
     restart: always
     tty: true
     depends_on:
       - mongodb
     environment:
       - MONGODB_PORT_27017_TCP_ADDR=mongodb
+      - HOME=/app
 
   mongodb:
     image: mongo:latest
     volumes:
-      - ./data/db:/data/db
+      - data:/data/db
     command: mongod --smallfiles --bind_ip=0.0.0.0 --logpath=/dev/null
     ports:
       - 27017:27017
 
+volumes:
+  data: {}
```

Comment:
I'd  actually argue against this, as I'd prefer to have the data local in the same directory. Thoughts?

Reply:
The benefit of a named volume gives me the ability to access the volume from another heroku container. It also makes it a bit more protected from accidental deletions when a user deletes his directory.

Reply:
I understand the heroku use case, but I can't really get behind the accidental deletion: if I delete the zenbot dir I would really also want to delete all DB data, and not have that linger somewhere invisible to me.